src/QuDynamics.jl

```diff
@@ -1,6 +1,8 @@
 module QuDynamics
     using QuBase
     using Compat
+    Pkg.checkout("ODE")
```

Comment:
I don't know if one can enforce `REQUIRE` to load the current master of a package? If not you can try to put `Pkg.checkout("ODE")` after `Pkg.build("QuDynamics");` in `.travis.yml` (the latter installs the latest tagged version ODE.jl and then the former switches it to the `master` branch).